loopback.cc

```diff
@@ -80,6 +92,10 @@ int main(int argc, char* argv[]) {
 	}
 
 	int fdwr = loopback_init(video_device,FRAME_WIDTH,FRAME_HEIGHT);
+	if(fdwr < 0) {
+		printf("Failed to initialize output device %s\n", video_device);
+		exit(1);
+	}
 
```

Comment:
As above, consistent output to `stderr` and line number, or unique descriptive message for identification.